ci/DevOps/local/test/build/nix/project-m36.nix

```diff
@@ -0,0 +1,72 @@
+{ mkDerivation, aeson, attoparsec, base, base64-bytestring, binary
+, blaze-html, bytestring, Cabal, cassava, conduit, containers
+, criterion, cryptohash-sha256, data-interval, deepseq
+, deepseq-generics, deferred-folds, directory, distributed-process
+, distributed-process-async, distributed-process-client-server
+, distributed-process-extras, either, exceptions, extended-reals
+, filepath, foldl, ghc, ghc-paths, Glob, gnuplot, hashable
+, hashable-time, haskeline, http-api-data, http-types, HUnit
+, list-t, megaparsec, monad-parallel, MonadRandom, mtl, network
+, network-transport, network-transport-tcp, old-locale
+, optparse-applicative, parallel, parser-combinators, path-pieces
+, QuickCheck, quickcheck-instances, random, random-shuffle
+, resourcet, rset, scotty, semigroups, stdenv, stm, stm-containers
+, template-haskell, temporary, text, time, transformers, unix
+, unordered-containers, uuid, vector, vector-binary-instances
+, websockets, zlib
+}:
+mkDerivation {
+  pname = "project-m36";
+  version = "0.7";
+  src = /home/chenjf/projects/project-m36/ci/DevOps/local/test/build/../../../../..;
```

Comment:
It won't because each time it's built the ci framework will call `cabal2nix` to generate the nix file and the src path will be updated to the correct one.